internal/pkg/deploy/cloudformation/cloudformation.go

```diff
@@ -139,7 +144,7 @@ func (cf CloudFormation) waitForStackCreation(stackConfig stackConfiguration) (*
 		StackName: aws.String(stackConfig.StackName()),
 	}
 
-	if err := cf.client.WaitUntilStackCreateComplete(describeStackInput); err != nil {
+	if err := cf.client.WaitUntilStackCreateCompleteWithContext(context.Background(), describeStackInput, cf.waiter); err != nil {
```

Comment:
Nice, I think with this change we'll reach the `MaxAttemp` faster:
https://github.com/aws/aws-sdk-go/blob/eb75dc68e5eba8519331d5de0c070883d5e75941/aws/request/waiter.go#L190-L192
120*3sec/60sec = 6 mins, so if one of our operations take longer than 6 mins it's going to timeout compared to the default of 1h.
That's probably ok! Just noting it here :), we could increase it with [WithWaiterMaxAttempts](https://docs.aws.amazon.com/sdk-for-go/api/aws/request/#WithWaiterMaxAttempts)

Reply:
NO that's a *really* good catch. I'll update the max attempts.

Reply:
Updated ^o^